simulator/Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: generate
-generate: vendor
-	go generate ./... && \
+generate: format
```

Comment:
now the mock files are checked by gci. So, mocks need to be formated after generated.